gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -259,44 +273,63 @@ public int available() throws IOException {
   }
 
   private void readAPITrace(
```

Comment:
what about have a default value for log level and provide a way to override it for specific APIs?